api/openapi.yaml

```diff
@@ -2277,6 +2326,11 @@ components:
             - aarch64
             - x86_64
           type: string
+        metadata:
+          description: An object of key/value associated with an Image
+          type: object
```

Comment:
Doesn't this need to be an array? Not a single object?

Reply:
Resolved via slack and subsequent simplification of patching.